Directory.Build.props

```diff
@@ -168,6 +168,11 @@
 
   <Import Project="$(RepositoryEngineeringDir)RuntimeIdentifier.props" />
 
+  <PropertyGroup>
+    <!-- Microsoft.NET.Sdk.IL SDK defaults to the portable host rid. Match it to the SDK RID (so source-build scenarios use the same RID as the host SDK). -->
+    <MicrosoftNetCoreIlasmPackageRuntimeId>$(NETCoreSdkRuntimeIdentifier)</MicrosoftNetCoreIlasmPackageRuntimeId>
```

Comment:
Previously `NETCoreSdkRuntimeIdentifier` was not yet set when `Directory.Build.props` got imported. It seems this has changed.

Some rid calculations were due to the lack of `NETCoreSdkRuntimeIdentifier`.

Reply:
I'll make sure to validate that this is available at this time (and move it later if needed).